UnitySDK/Assets/ML-Agents/Examples/FoodCollector/Scripts/FoodCollectorAgent.cs

```diff
@@ -150,7 +151,7 @@ public void MoveAgent(float[] act)
         {
             var myTransform = transform;
             myLaser.transform.localScale = new Vector3(1f, 1f, m_LaserLength);
-            var position = myTransform.TransformDirection(RayPerception3D.PolarToCartesian(25f, 90f));
+            var position = myTransform.TransformDirection(RayPerceptionSensor.PolarToCartesian(25f, 90f));
```

Comment:
Should this be `3DRayPerceptionSensor` to show this is 3D?

Reply:
Should PolarToCartesian live in some Utility class? Should the Debug.DrawRay be part of the sensor instead ?

Reply:
Let me try to clean some of this up a bit. I'd lean towards not doing any trig functions at runtime, and either save a quaternion "base" and "increment", or storing the local space vectors on the sensor.

Reply:
🤦‍♂ This should just be "25.0 * myTransform.TransformDirection(transform.forwards)" since the angle is hardcoded to 90

Reply:
Cleaned this part up. It had no need for any RayPerception* code here. Note this is "game logic" inside MoveAgent(), not actually related to observations.